bot/cogs/queue.py

```diff
@@ -32,41 +14,36 @@ def __init__(self, bot):
         """ Set attributes. """
         self.bot = bot
         self.guild_queues = {}  # Maps Guild: QQueue
+        self.last_queue_msgs = {}
 
-    @commands.Cog.listener()
-    async def on_ready(self):
-        """ Initialize an empty list for each guild the bot is in. """
-        for guild in self.bot.guilds:
-            if guild not in self.guild_queues:  # Don't add empty queue if guild already loaded
-                self.guild_queues[guild] = QQueue()
-
-    @commands.Cog.listener()
-    async def on_guild_join(self, guild):
-        """ Initialize an empty list for guilds that are added. """
-        self.guild_queues[guild] = QQueue()
-
-    @commands.Cog.listener()
-    async def on_guild_remove(self, guild):
-        """ Remove queue list when a guild is removed. """
-        self.guild_queues.pop(guild)
-
-    def queue_embed(self, guild, title=None):
+    async def queue_embed(self, guild, title=None):
         """ Method to create the queue embed for a guild. """
-        queue = self.guild_queues[guild]
+        queued_ids = await self.bot.db_helper.get_queued_users(guild)
+        guild_data = await self.bot.db_helper.get_guild(guild)
+        capacity = guild_data['capacity']
 
         if title:
-            title += f' ({len(queue.active)}/{queue.capacity})'
+            title += f' ({len(queued_ids)}/{capacity})'
 
-        if queue.active != []:  # If there are users in the queue
-            queue_str = ''.join(f'{e_usr[0]}. {e_usr[1].mention}\n' for e_usr in enumerate(queue.active, start=1))
-        else:  # No users in queue
+        if len(queued_ids) == 0:  # If there are users in the queue
```

Comment:
The comments on this if statement are the wrong way around

Reply:
Will fix this.